src/chains/filecoin/filecoin/src/blockchain.ts

```diff
@@ -255,6 +270,10 @@ export default class Blockchain extends Emittery.Typed<
         );
       }
     }
+
+    fileStream.close();
```

Comment:
I think this just needs to be a `finally` block in order to get the :+1: from me!